clusterbuster

```diff
@@ -2427,6 +2440,65 @@ EOF
     done
 }
 
+function create_vm_deployment() {
+    local namespace=$1
+    local instance=$2
+    local -i replicas=$4
+    local -i replica=0
+    local depname="${namespace}-${workload}-${instance}"
+    while (( replica++ < replicas )) ; do
+        local name="${depname}-${replica}"
+        local vmname=$(mkpodname "$name")
+        create_object -n "$namespace" "$name" <<EOF
+apiVersion: kubevirt.io/v1alpha3
+kind: VirtualMachine
+metadata:
+  labels:
+    app: "$name"
+    kubevirt-vm: "$vmname"
+  name: "$vmname"
+$(indent 2 standard_deployment_metadata_yaml "$namespace" client)
+$(indent 2 standard_pod_metadata_yaml "$namespace" client)
+$(create_vm_spec -n "$vmname")
+EOF
+    done
+}
+
+function create_vm_spec() {
+    local vmname="fedora"
+    while getopts 'n:' opt; do
+        case "$opt" in
+            n) vmname=$OPTARG ;;
+        esac
+    done
+    [[ -n "$vmname" ]] || fatal "vmname is required"
+    cat <<EOF
+spec:
+  running: true
+  template:
+    metadata:
+      labels:
+        kubevirt-vm: "$vmname"
```

Comment:
Use `standard_labels_yaml` to ensure that everything gets labeled in a way that it can be identified by cleanup.  You can add the `kubevirt-vm` label after that.

Reply:
> Also update _do_cleanup() to delete VMs.

Since VM is a long running process, how do I trigger a normal  `cleanup` process where oc commands actually gets invoked?

Reply:
When clusterbuster completes, it by default cleans up.  What's the deployment model you're looking at here?

Reply:
Is this fixed now?

Reply:
Yeah cleanup is working now.